WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -457,10 +457,14 @@ protected void onCreate(Bundle savedInstanceState) {
             return;
         }
 
+        if (savedInstanceState == null) {
+            // Bump the stat the first time the editor is opened.
+            PostUtils.trackOpenPostAnalytics(mPost, mSite);
```

Comment:
There is a `savedInstanceState == null` branch earlier in the `onCreate` method of the `EditPostActivity` class [here](https://github.com/wordpress-mobile/WordPress-Android/blob/e88dc3f19defc1c059da2d99e081d82661bfb31c/WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java#L376).  Can we move the `PostUtils.trackOpenPostAnalytics(mPost, mSite);` statement there to keep all the code executed when the activity it first created in one place?

Reply:
Just above this block are checks for a null site and null post.  I'd like to avoid bumping the stat in cases where we'd otherwise bail out of the activity due to an invalid post or site.  If we bump the stat earlier in the method it looks like we'd risk that?